src/client/sandbox/node/live-node-list/dom-mutation-tracker.ts

```diff
@@ -16,12 +16,12 @@ class DOMMutationTracker {
         });
     }
 
-    _updateVersion (tagName) {
+    _updateVersion (tagName: string) {
         if (tagName in this._mutations)
             this._mutations[tagName].increment();
     }
 
-    _processElement (el) {
+    _processElement (el: any) {
```

Comment:
The `any` type is default type. Why do we need to specify it?

Reply:
![image](https://user-images.githubusercontent.com/4133518/61947921-1d773500-afaf-11e9-821e-04133e49ba89.png)